### ISSUE TITLE
Warn the user if the DS app is installed on the phone

### ISSUE THREAD

**Issue by ItsHarper** [OpenFTC/OpenFTC-app#8]
Give them the option to be sent to an uninstallation page

**Comment by WardBenjamin** [comment]
Just as a note, there's some code to detect if the DS is installed in the Inspection Activity. From there, it should be as easy as follows:

```
final String appPackageName = getPackageName(); // getPackageName() from Context or Activity object
try {
    startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("market://details?id=" + appPackageName)));
} catch (android.content.ActivityNotFoundException anfe) {
    startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("https://play.google.com/store/apps/details?id=" + appPackageName)));
}
```

**Comment by Windwoes** [comment]
Thanks; I actually already wrote the code for doing that yesterday, just haven't PRed yet

**Comment by Windwoes** [comment]
Ok, this was merged in #20, closing this
